pixart.js

```diff
@@ -1 +1,14 @@
+for(var i = 0; i < 8000; i++) {
+ var square = document.createElement("div");
+ square.setAttribute("class", "square");
+ document.body.appendChild(square);
+}
 
+var draw = document.querySelectorAll(".square");
```

Comment:
Make sure to pick meaningful variable names. I'd choose something like 'squares' here since it's a list of squares (or maybe `pixels` would be good).